kill.go

```diff
@@ -23,7 +22,7 @@ Where "<container-id>" is the name for the instance of the container and
 EXAMPLE:
 For example, if the container id is "ubuntu01" the following will send a "KILL"
 signal to the init process of the "ubuntu01" container:
-	 
+
```

Comment:
Sorry about that, but that's the only place with EOL whitespace in all the .go files in this repo, and in my mind it does not justify a separate commit to fix it, so I shoved it in here.

Reply:
All good.